svn/trunk/SlickEngagement_Plugin.php

```diff
@@ -452,7 +398,7 @@ public function addSlickPageHeader()
                 echo "if (window.slickAbTestResult(" . $enabledPercent . ")) {\n";
             }
             echo "/* Slickstream Engagement Suite Embedder */\n";
-            echo '"use strict";(async(e,t)=>{if(location.search.indexOf("no-slick")>=0){return}let o;const c=()=>(performance||Date).now();const a=window.$slickBoot={rt:e,_es:c(),ev:"2.0.0",l:async(e,t)=>{try{let a=0;if(!o&&"caches"in self){o=await caches.open("slickstream-code")}if(o){let n=await o.match(e);if(!n){a=c();await o.add(e);n=await o.match(e);if(n&&!n.ok){n=undefined;o.delete(e)}}if(n){const e=n.headers.get("x-slickstream-consent");return{t:a,d:t?await n.blob():await n.json(),c:e||"na"}}}}catch(e){console.log(e)}return{}}};const n=e=>new Request(e,{cache:"no-store"});const i=n(`${e}/d/page-boot-data?${innerWidth<=600?"mobile&":""}site=${t}&url=${encodeURIComponent(location.href.split("#")[0])}`);let{t:s,d:l,c:d}=await a.l(i);if(l){if(l.bestBy<Date.now()){l=undefined}else if(s){a._bd=s;a.c=d}}if(!l){a._bd=c();const e=await fetch(i);const t=e.headers.get("x-slickstream-consent");a.c=t||"na";l=await e.json()}if(l){a.d=l;let e=l.bootUrl;const{t:t,d:o}=await a.l(n(e),true);if(o){a.bo=e=URL.createObjectURL(o);if(t){a._bf=t}}else{a._bf=c()}const i=document.createElement("script");i.src=e;document.head.appendChild(i)}else{console.log("[Slick] Boot failed")}})' . "\n";
+            echo '"use strict";(async(e,t)=>{if(location.search.indexOf("no-slick")>=0){return}let o;const c=()=>(performance||Date).now();let a=window.$slickBoot=window.$slickBoot||{};a.rt=e;a._es=c();a.ev="2.0.0";a.l=async(e,t)=>{try{let a=0;if(!o&&"caches"in self){o=await caches.open("slickstream-code")}if(o){let i=await o.match(e);if(!i){a=c();await o.add(e);i=await o.match(e);if(i&&!i.ok){i=undefined;o.delete(e)}}if(i){const e=i.headers.get("x-slickstream-consent");return{t:a,d:t?await i.blob():await i.json(),c:e||"na"}}}}catch(e){console.log(e)}return{}};const i=e=>new Request(e,{cache:"no-store"});if(!a.d){const o=i(`${e}/d/page-boot-data?site=${t}&url=${encodeURIComponent(location.href.split("#")[0])}`);let{t:n,d:s,c:l}=await a.l(o);if(s){if(s.bestBy<Date.now()){s=undefined}else if(n){a._bd=n;a.c=l}}if(!s){a._bd=c();const e=await fetch(o);const t=e.headers.get("x-slickstream-consent");a.c=t||"na";s=await e.json()}if(s){a.d=s;a.s="embed"}}if(a.d){let e=a.d.bootUrl;const{t:t,d:o}=await a.l(i(e),true);if(o){a.bo=e=URL.createObjectURL(o);if(t){a._bf=t}}else{a._bf=c()}const n=document.createElement("script");n.src=e;document.head.appendChild(n)}else{console.log("[Slick] Boot failed")}})' . "\n";
```

Comment:
Are we not removing the `await caches.open` (browser application cache) part in this release?

Reply:
not for this release please